src/chocolatey/infrastructure.app/templates/ChocolateyReadMeTemplate.cs

```diff
@@ -80,77 +82,77 @@ You should read up on the Shim Generation section to familiarize yourself
- * IS_REMOTE = Is the user in a remote session?
+ * {EnvironmentVariables.Package.OsIs64Bit} = This may not return correctly - it may depend on the process the app is running under
+ * {EnvironmentVariables.Package.ChocolateyProductVersion} = the version of Choco that may match {EnvironmentVariables.Package.ChocolateyVersion} but may be different - based on git describe
+ * {EnvironmentVariables.Package.IsAdmin} = Is the user an administrator? But doesn't tell you if the process is elevated.
+ * {EnvironmentVariables.Package.IsRemote} = Is the user in a remote session?
 
 #### Not Useful Or Anti-Pattern If Used
 
- * ChocolateyInstallOverride = Not for use in package automation scripts. Based on `--override-arguments` being passed.
- * ChocolateyInstallArguments = The installer arguments meant for the native installer. You should use chocolateyPackageParameters instead. Based on `--install-arguments` being passed.
- * ChocolateyIgnoreChecksums - Was `--ignore-checksums` passed or the feature `checksumFiles` turned off?
- * ChocolateyAllowEmptyChecksums - Was `--allow-empty-checksums` passed or the feature `allowEmptyChecksums` turned on?
- * ChocolateyAllowEmptyChecksumsSecure - Was `--allow-empty-checksums-secure` passed or the feature `allowEmptyChecksumsSecure` turned on?
- * ChocolateyChecksum32 - Was `--download-checksum` passed?
- * ChocolateyChecksumType32 - Was `--download-checksum-type` passed?
- * ChocolateyChecksum64 - Was `--download-checksum-x64` passed?
- * ChocolateyChecksumType64 - Was `--download-checksum-type-x64` passed?
+ * {EnvironmentVariables.Package.ChocolateyInstallOverride} = Not for use in package automation scripts. Based on `--override-arguments` being passed.
+ * {EnvironmentVariables.Package.ChocolateyInstallArguments} = The installer arguments meant for the native installer. You should use {EnvironmentVariables.Package.ChocolateyPackageParameters} instead. Based on `--install-arguments` being passed.
+ * {EnvironmentVariables.Package.ChocolateyIgnoreChecksums} - Was `--ignore-checksums` passed or the feature `checksumFiles` turned off?
+ * {EnvironmentVariables.Package.ChocolateyAllowEmptyChecksums} - Was `--allow-empty-checksums` passed or the feature `allowEmptyChecksums` turned on?
+ * {EnvironmentVariables.Package.ChocolateyAllowEmptyChecksumsSecure} - Was `--allow-empty-checksums-secure` passed or the feature `allowEmptyChecksumsSecure` turned on?
+ * {EnvironmentVariables.Package.ChocolateyChecksum32} - Was `--download-checksum` passed?
+ * {EnvironmentVariables.Package.ChocolateyChecksumType32} - Was `--download-checksum-type` passed?
+ * {EnvironmentVariables.Package.ChocolateyChecksum64} - Was `--download-checksum-x64` passed?
+ * {EnvironmentVariables.Package.ChocolateyChecksumType64} - Was `--download-checksum-type-x64` passed?
  * ChocolateyPackageExitCode - The exit code of the script that just ran - usually set by `Set-PowerShellExitCode`

```

Comment:
Is there a reason that this one hasn't been done?

Reply:
We don't set this in the executable itself, it may be set by one of the PowerShell scripts, but we aren't aware of it at the moment, so I left it as is.

Reply:
Wait a minute...

We have `EnvironmentVariables.Package.ChocolateyExitCode`, so shouldn't that be being used here?

Reply:
We do have that one, but it is not the one that is documented in the template, it is a different one.